### ISSUE TITLE
Where are the bam files?

### ISSUE THREAD

**Issue by millerh1** [vallotlab/scChIPseq#1]
Hello!

I am unable to locate the BAM files for the ChIP-Seq experiments. It says in the README:

"To run the analysis for the 4 datasets in the paper first download all the matrices and bam files in the repo root."

These files do not seem to be included in the GEO entry or mentioned anywhere in the article. Where can the bam files be downloaded from? I am really hoping to avoid re-analyzing the fastq files if possible!

I saw that you may download the 10X bam files for the scRNA-Seq but I don't see any option to do so for the scChIP-Seq experiment. 

Thank you!

**Comment by gamazeps** [comment]
Hi Henry !

The bam files are only needed if you want to do peak calling, which may not bee what you are looking for.
Launching the command without the `b1` or `b2` arguments should work fine, so you can analyse the data without them (at least for now).

We are looking into why they are not in the GEO.

Cheers

**Comment by gamazeps** [comment]
Hi again Henry !

The bams can be found on this [figshare](https://figshare.com/s/fb04c2b17b234aa9d5eb)

Cheers,
Felix

**Comment by millerh1** [comment]
Thank you so much for the response!!

That worked for me and I was able to successfully run the code in your repository. I was just wondering if you happened to have an mm10 alignment file for the HBCx-22 and HBCx-22 TamR samples as well? I noticed that it wasn't in the run_paper.sh commands and wasn't in figshare, but that there were in count tables in GEO for those samples (e.g. GSM3290892 supplementary files). Thank you again for looking into this!
